src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -316,6 +330,18 @@ public static void set(GWC instance) {
         GWC.INSTANCE = instance;
     }
 
+    /**
+     * Only to aid in unit testing for the places where a mock GWC mediator and GeoServer
+     * environment synchhronizer are needed
+     *
+     * @param instance the GWC mediator instance
+     * @param gwcSynchEnv the GeoServer environment synchronizer instance
+     */
+    public static void set(GWC instance, GWCSynchEnv gwcSynchEnv) {
```

Comment:
Thinking out loud, is the other ``set(GWC)`` method still getting any usage? (I'm asking because I see tests using it being switched to this new method).

Reply:
It was in other places, but cleaner to switch to two argument form everywhere.  Fixed.